tests/phpunit/tests/st/strategy/shortcode/fixtures/content-wrapper/empty-strings-between-2-shortcodes-wrapped.html

```diff
@@ -0,0 +1,3 @@
+[shortcode_A]	[av_notification title='Note']
```

Comment:
@strategio What is `shortcode_A`? I assume it's an unknown shortcode. Mabye `[unknown_shortcode_A]` would be clearer.

Reply:
No @brucepearson, the `shortcode_A` is a valid one. In this content, we don't have any regular text to translate (and we don't want to handle the whitespaces, tabs, ...).

Reply:
If `shortcode_A` was not a registered shortcode, we would wrap it.